README.md

```diff
@@ -239,6 +239,7 @@ Table of Contents
   * [KSoft.Si](https://api.ksoft.si/) — Free lyrics api chiefly aimed for discord bots.Also provides an extensive library of images and user data
   * [Lightly](https://www.lightly.ai/) — Improve your machine learning models by using the right data. Use datasets of up to 1'000 samples for free.
   * [MailboxValidator](https://www.mailboxvalidator.com) — Email verification service using real mail server connection to confirm valid email. Free API plan has 300 verifications per month.
+  * [MetalpriceAPI.com](https://metalpriceapi.com) - Live (Spot) and Historical Precious Metals and Currency Rates API. Also provides currency conversion API. Free developer plan available with 100 requests/month.
```

Comment:
100 requests/month? I dont think that's useful you can barely do any kind of testing with that. So I think I'll pass on this one